config/settings.yml

```diff
@@ -1,3 +1,4 @@
+LIBRARY_WEBSITE_JSON_API_URL: 'https://library.stanford.edu/bento?keys=%{q}'
```

Comment:
I think since we've done this, our configuration approach/pattern has shifted a bit.  Would there be interest in shifting this config to match that? e.g. something like

```yaml
LIBRARY_WEBSITE:
  API_URL: ...
```

Also, to match how some of our other searchers are working, I would expect to see `library_website_api` in the list of ENABLED_SEARCHERS in `test.yml` (although that may be because of how we'll transition the libweb searchers, not sure).

Reply:
I ticketed this here: https://github.com/sul-dlss/sul-bento-app/issues/360

but was kind of waiting until we shipped to rework everything so we don't need to end up changing the legacy searcher that we are going to end up removing. 🤷

Reply:
I'm not suggesting that we do this for the legacy searcher, just the new one.

Reply:
Totally.. and the reason I was hesitant to do for the new one until the follow up was then the config would be split in two different ways because the new json searcher also uses a legacy config `Settings.LIBRARY_WEBSITE_QUERY_API_URL`.

🤷

Reply:
That would seem to me like a reason to put the new searcher's configs into the new pattern and not use that same config as the old searcher (so that we don't accidentally break the new searcher while cleaning up the old one), but it's def. not a big deal.  👍

Reply:
Got it.. I can make that happen.

Reply:
Does bb982aa8dbd121e6050327ef4fa6a8ebf6af394e work out for now?

Reply:
![Perfection](https://media1.tenor.com/images/a7031e8ce4a66c2aaafd87aca298290a/tenor.gif?itemid=7851132)